src/cmd/api/metrics/metrics.go

```diff
@@ -8,13 +8,13 @@ import (
 
 func InitMetrics() {
 	if err := prometheus.Register(TotalRequests); err != nil {
-		log.Error("Unable to register metrics.TotalRequests prometheus")
+		log.Warning("Unable to register metrics.TotalRequests prometheus")
```

Comment:
While you are at it, knowing why we have those error can be cool

Reply:
Fixed : https://github.com/prometheus/client_golang/issues/716